packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -179,15 +363,11 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
     }
   }
 
-  function handleOnInputValueChange(inputValue, { type }) {
+  function handleOnInputValueChange(inputValue: string) {
```

Comment:
I've been looking for this mysterious `type` attribute and haven't been able to find it anywhere for this handler… My hunch is that someone has mistaken this with some kind of `stateChange` handler, where it might exist and since this evidently wasn't checked by anything it slipped by

Reply:
I see your [comment above](https://github.com/carbon-design-system/carbon/pull/13615/files#diff-a1249d25d8548f5c2c1a100e428df6fb0f881e7f213b96994153fc4aede57f33R146-R149) too. This mismatch could be because we're using v5.x downshift and latest is v7.x?

Reply:
Dunno… your guess is as good as mine, I think.

What I know for certain is that Downshift upgrade is not a joke… also fixing one handler the other way around can quickly spiral to a huge refactor due to many call sites and the messy hooks.